fc/jira/issue.py

```diff
@@ -114,9 +118,11 @@ def get_issue(cls, issue_id: str, auth: Auth) -> 'Issue':
             raise TaskException('Invalid issue key {}'.format(issue_id)) from exception
 
         type = issue_json['fields']['issuetype']['name']
-        if type != 'Task' and type != 'Triage Task':
+        if type != 'Task' and type != 'Story' and type != 'Triage Task':
```

Comment:
Did you try other stories in other spaces, such as `QPPCT-4`, such that it would create an `Issue` and not a `BacklogStory`?

Reply:
I actually added a `program` field to `Issue` so that we can interrogate it.  Since we look at the actual jira type to determine which Python type to instantiate, issues from other projects will create `BacklogTask` or `BacklogStory` objects.  So I added a program check to the `backlog score` functionality.  Other functionality should do a check for program as well if necessary.